Assets/TouchKit/Recognizers/TKSwipeRecognizer.cs

```diff
@@ -141,7 +148,7 @@ private bool checkForSwipeCompletion(TKTouch touch)
         float idealDistanceCM = idealDistance / TouchKit.instance.ScreenPixelsPerCm;
 
         // if the distance moved in cm was less than the minimum,
-        if (idealDistanceCM < this._minimumDistance)
+        if (idealDistanceCM < this._minimumDistance || idealDistanceCM > this.maximumDistance)
```

Comment:
This line is failing because it should be `this._maximumDistance`.

Reply:
This is fixed